Dockerfile

```diff
@@ -1,16 +1,29 @@
 # vim:set ft=dockerfile:
-FROM vertexproject/synapse-base-image:py36
 
-ENV SYN_DMON_LOG_LEVEL="WARNING"
+# Example Usage
+#
+# To Build:
+#   docker build -t vertexproject/synapse:localdev .
+#
+# To Run a example Cortex on port 47322:
+#   docker run -it -p47322:47322 vertexproject/synapse:localdev
+#
+# To create and run your own dmon (do not create the dmon dir inside synapse dir)
+#   python -m synapse.tools.deploy --listen tcp://0.0.0.0:47322 cortex core dmon_dir
+#   docker run -it -p47322:47322 -v "$(pwd)"/dmon_dir:/syndata/dmon_dir vertexproject/synapse:localdev
+
+FROM vertexproject/synapse-base-image:py36
 
-COPY . /root/git/synapse
-RUN mkdir /syndata \
- && cd /root/git/synapse && python setup.py install \
- && cp synapse/docker/cortex/ram_dmon.json /syndata/dmon.json
+COPY . /root/git/synapse/
+RUN \
+    # Install Synapse
+    cd /root/git/synapse && \
+    python setup.py develop && \
+    # Create a default dmon directory
+    mkdir -p /syndata && cd /syndata && \
+    python -m synapse.tools.deploy --listen tcp://0.0.0.0:47322 cortex core dmon_dir
 
 VOLUME /syndata
 VOLUME /root/git/synapse
 
-WORKDIR /root/git/synapse
-EXPOSE 47322
```

Comment:
Do we want to expose this port by default still? it's one less configuration step for a person to do themselves?

Reply:
I'll add this back